R/diagnostics.R

```diff
@@ -19,6 +19,10 @@
 #'   tolerance set by {tol}, FALSE otherwise.
 #' @export
 isTimestepRegular <- function(dates, hist=TRUE, tol=.Machine$double.eps^0.5, handler=stop) {
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes=TimeInterval=geom_histogram=xlab=NULL
```

Comment:
miguel wrote: ok, the import works @importFrom ggplot2 ggplot aes geom_histogram xlab 
I'm not sure where TimeInterval comes from though, I should change it to ??.TimeInterval ?

(if you start a comment with "Add a line note" button, we can maintain several conversation threads in one PR)